### DISCUSSION TITLE
How to Train Model or can say Finetune on Custom Dataset means own voice samples Step by Step

### DISCUSSION THREAD

**Discussion by editorlue** [SWivid/F5-TTS#119] [Q&A]
How to Train Model or can say Finetune on Custom Dataset means own voice samples Step by Step.

Please Include things What are the changes need to be done after cloning for custom dataset provide all steps and tell how to fix errors

**Comment by kunibald413** [comment]
https://github.com/SWivid/F5-TTS/discussions/57#discussioncomment-10959029
